README.md

```diff
@@ -48,6 +48,20 @@ There are currently three main scripts available:
 
 For all of these, it's assumed that you are running from the `scripts/` subdirectory.
 
+### Running Tests: via Docker 
+
+You can easily run the tests included in this repo. These tests should be run, and pass, before submitting an update to the repo. 
+The tests are packaged in a `docker-compose` file for easy execution. Docker and Docker Compose must be installed to run tests using this method. 
+See the links at the end of this section for installation links.
+
+To run the tests, from the repo root, issue this command:
+`docker-compose -f docker-compose-test.yml run test`
+
+
+Here is more information on installing docker and docker-compose.
+* [Installing Docker](https://docs.docker.com/engine/installation/)
+* [Installing Docker Compose](https://docs.docker.com/compose/install/)
+
```

Comment:
Excellent, thank you for adding this documentation on how to use the tests via Docker.